packages/suite/package.json

```diff
@@ -99,6 +100,7 @@
         "core-js-bundle": "3.8.2",
         "cross-env": "^7.0.3",
         "fake-indexeddb": "^3.1.2",
+        "intersection-observer": "^0.12.0",
```

Comment:
we already have `react-intersection-observer` in dependency, do we need another lib?

Reply:
it's there just for unit tests, so it's only dev-depencency, see https://github.com/thebuilder/react-intersection-observer#testing